kuiper-game/kuiper-game.csproj

```diff
@@ -14,5 +14,8 @@
     <Content Include="Data\*.solarsystem">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
```

Comment:
Maybe delete the old wildcard include if we're including the single file as well?
```suggestion

```